airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/reports.py

```diff
@@ -25,6 +25,8 @@
 class ConnectorReport(Report):
     """A dataclass to build connector test reports to share pipelines executions results with the user."""
 
+    ENABLE_PR_COMMENTS: bool = False
```

Comment:
Im a bit apprehensive of leaving unused code in the code base.

Code is cattle not a pet ([ref](https://www.hava.io/blog/cattle-vs-pets-devops-explained))

I would argue that we remove it, unless theres a reason to keep it around that I may be missing.

Reply:
When we will consider running the CI on community PRs I think having these comments back can be a good way to expose high-level results to the community contributors. HTML reports are currently private to not overly expose logs.